aptos-move/mvhashmap/src/versioned_aggregators.rs

```diff
@@ -33,11 +35,11 @@ enum AggregatorEntry {
     // of the same transaction, the delta may also be kept in the entry. This is useful
     // if speculative execution aborts and the entry is marked as estimate, as the delta
     // may be used to avoid waiting on the Estimate entry. More in comments below.
-    Value(u128, Option<DeltaOp>),
-    Delta(DeltaOp),
+    Value(AggregatorValue, Option<AggregatorApplyChange>),
+    // Applies the change on top of the previous entry.
     // Applies the delta on top of a 'snapshot' of an aggregator with a given ID (at
     // a corresponding, i.e. one less than where the given entry is stored, version).
-    Snapshot(AggregatorID, DeltaOp),
+    Apply(AggregatorApplyChange),
```

Comment:
nit: Application?

Reply:
Why? :) It's much longer, and it goes into AggregatorChange::Apply as well as AggregatorApplyChange. 

maybe if there is a better word instead